lib/gollum-lib/macro/video.rb

```diff
@@ -0,0 +1,9 @@
+module Gollum
+  class Macro
+    class Video < Gollum::Macro
+      def render (fname)
+        "<video width=\"100%\" height=\"100%\" src=\"#{fname}\" controls> HTML5 video is not supported on this Browser.</video>"
```

Comment:
Sweet, that's a really simple implementation. I guess almost the exact same code could be used for the audio Macro.